src/care/host_device_map.h

```diff
@@ -399,7 +405,13 @@ namespace care {
          }
          
          // copy constructor 
-         host_device_map(host_device_map const & other) noexcept = default;
+         host_device_map(host_device_map const & other) noexcept {
```

Comment:
This one is host only, too, so it can be reverted back to `= default;`

Reply:
Reverted